.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+Thanks for raising a Spring Boot issue. What sort of issue are you raising?
+
+Question
+
+Please ask questions about how to use something, or to understand why something isn't
+working as you expect it to, on Stack Overflow using the spring-boot tag.
```

Comment:
using the `spring-cloud` tag I guess. We also have tags for the concrete projects (e.g. `spring-cloud-sleuth`)